src/pages/settings/Profile/ProfilePage.js

```diff
@@ -149,6 +161,14 @@ class ProfilePage extends Component {
             return;
         }
 
+        if (this.state.avatarImage) {
+            PersonalDetails.setAvatar(this.state.avatarImage);
+        }
+
+        if (!this.props.myPersonalDetails.avatar.includes('/images/avatars/avatar') && !this.state.avatarImage) {
```

Comment:
Should we add a comment for this? It isn't obvious what `/images/avatars/avatar` is

Reply:
I've added a comment, let me know if you need to add something more.

Reply:
Thanks